src/frontend/package.json

```diff
@@ -25,6 +37,7 @@
     "@fortawesome/react-fontawesome": "0.2.0",
     "@uiw/react-md-editor": "3.24.1",
     "breakpoint-sass": "3.0.0",
+    "dotenv": "16.3.1",
```

Comment:
If we update to node 20.6, dotenv is no longer required to access process.env

Reply:
are we planning on moving soon?

Reply:
We're already on 20.5 I think so not hard to update.